PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -298,7 +298,11 @@ def miniAOD_customizeCommon(process):
     run2_miniAOD_94XFall17.toModify(process.electronMVAValueMapProducer,
                                      keysForValueMaps = cms.InputTag('reducedEgamma','reducedGedGsfElectrons'),
                                      src = cms.InputTag("gedGsfElectronsFrom94XTo106X"))
-
+    from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+    pp_on_AA_2018.toModify(task, func=lambda t: t.add(process.gedGsfElectronsFrom94XTo106XTask))
+    pp_on_AA_2018.toModify(process.electronMVAValueMapProducer,
+                                     keysForValueMaps = cms.InputTag('reducedEgamma','reducedGedGsfElectrons'),
+                                     src = cms.InputTag("gedGsfElectronsFrom94XTo106X"))
```

Comment:
```suggestion
                                     keysForValueMaps = 'reducedEgamma:reducedGedGsfElectrons',
                                     src = "gedGsfElectronsFrom94XTo106X")
```
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
No problem for 'src', but python complains if the type is not specified for 'keysForValueMaps'

Reply:
apparently the keysForValueMaps parameter did not already exist, so having to specify a type for it is appropriate